docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_fluid_loop_attached_to_chiller.md

```diff
@@ -13,10 +13,12 @@
 
 ## Logic:   
 - Set is_fluid_loop_attached_to_chiller equal to FALSE: `is_fluid_loop_attached_to_chiller = FALSE`  
-- For each chiller in B_RMR, save chiller to loop chiller dictionary: `for chiller_b in B_RMR.RulesetModelInstance.chillers: loop_chiller_dict[chiller_b.cooling_loop].append(chiller_b)`
-- Create an object associate with the cooling_system fluid loop associated with hvac_b: `fluid_loop_b = hvac_b.cooling_system[0].chilled_water_loop`
+- For each chiller in B_RMR, save the list of the associated primary and secondary loops: `for chiller_b in B_RMR.RulesetModelInstance.chillers:`   
+    - Add the secondary loops associated with the chiller system primary loop to a list of loops: `loop_list.extend(list(chiller_b.cooling_loop.child_loops))`  
```

Comment:
Do we need to check for scenario with tetriary etc. loops - i.e., where a loop included in the child_loops has child_loops of its own? That should never occure in the baseline, but may come up in the proposed design.